django_swagger_tester/utils.py

```diff
@@ -262,7 +261,8 @@ def get_path(self) -> str:
         path = f'{path[:starting_index]}{self.resolved_path.kwargs[parameter_name]}{path[starting_index + len(parameter):]}'
         self.parameterized_path = path
         self.counter += 1
-        return path
+
+        return self.replace_i18n_parameter(path)
```

Comment:
`self.replace_i18n_parameter(self.parameterized_path)` also needs to be returned when `self.counter == 0` (line 253). Currently, the i18n parameter only gets replaced when there is also another parameter in the path (you can test this by removing the `version` parameter of the `/api/v1/i18n` endpoint).

Reply:
Good catch 👏